src/wallet/feebumper.cpp

```diff
@@ -169,13 +167,30 @@ Result CreateRateBumpTransaction(CWallet& wallet, const uint256& txid, const CCo
     }
     const CWalletTx& wtx = it->second;
 
+    // Retrieve all of the UTXOs and add them to coin control
+    // While we're here, calculate the input amount
+    std::map<COutPoint, Coin> coins;
+    CAmount input_value = 0;
+    for (const CTxIn& txin : wtx.tx->vin) {
+        coins[txin.prevout]; // Create empty map entry keyed by prevout.
+    }
+    wallet.chain().findCoins(coins);
+    for (const CTxIn& txin : wtx.tx->vin) {
+        const Coin& coin = coins.at(txin.prevout);
+        assert(!coin.out.IsNull());
+        new_coin_control.SelectExternal(txin.prevout, coin.out);
```

Comment:
There is largely no difference, but since #24649, our pattern is to actually only use `SelectExternal` for external inputs, so I have changed this to do that.